javascript/src/main/pages/Search/Search.js

```diff
@@ -46,13 +47,24 @@ const Search = () => {
             console.log(`err=${err}`);
             return {quota:0};
         }
-    };
+    },[getToken])
 
     
     const [query, setQuery] = useState(emptyQuery);
     const [results, setResults] = useState(emptyResults);
     const [quota, setQuota] = useState(0);
 
+    useEffect(() => {
+        async function getQuota() {
+            const quotaInfo = await fetchQuota();
+            console.log("quotaInfo=",quotaInfo)
+            if(quotaInfo && quotaInfo.quota)
+                setQuota(quotaInfo.quota);
+        };
```

Comment:
This line not being covered by tests?  We are going to let that slide!!!  Sincerely, Prof. Conrad. Kthxbye.